### PR TITLE
update gbw model after transform API refactoring

### PR DESCRIPTION
**Pull request by eric-haibin-lin**
## Description ##
Tested on master branch with test ppl 43.72. 

### Essentials ###
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: gluonnlp/data/corpora/google_billion_word.py [hunk 1]
```diff
@@ -136,16 +136,16 @@ def __init__(self, segment='train', skip_empty=True, bos=None, eos=C.EOS_TOKEN,
                  root=os.path.join(_get_home_dir(), 'datasets', 'gbw')):
         self._archive_data = ('1-billion-word-language-modeling-benchmark-r13output.tar.gz',
                               '4df859766482e12264a5a9d9fb7f0e276020447d')
-        self._archive_vocab = ('gbw-b3e83215.zip',
-                               '9502eb728bf492a382b1cd57b374734ff82f8553')
+        self._archive_vocab = ('gbw-ebb1a287.zip',
+                               '63b335dcc27b6804d0a14acb88332d2602fe0f59')
```
Comment (eric-haibin-lin) [inline gluonnlp/data/corpora/google_billion_word.py]: This hash from @szha's second upload of `gbw-ebb1a287.zip`

**Comment by mli** [comment]
Job PR-294/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-294/1/index.html

**Comment by mli** [comment]
Job PR-294/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-294/2/index.html

**Comment by mli** [comment]
Job PR-294/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-294/3/index.html

**Pull request merged**